File: 27.RemoveElement/Python/ccc.py
nums = [3,2,1,3]
val = 3



def removeElement(nums, val):
    i = 0
    for x in nums:
        print()
        print(i)
        print(x)
        print(nums[i])
        print(nums)
        if x != val:
            print("*")
            nums[i]=x
            print(nums[i])
            i += 1
            print(i)
            print(nums)    
    nums = nums[:i]
    print("\n",nums)
    return i
    

print(removeElement(nums,val))

File: 27.RemoveElement/JS/ccc.js

var removeElement = function(nums, val) {
    let i = 0;
    // for (let x = 0; x< nums.length; x++){
    for (x in nums){
        if (nums[x] != val){
            console.log(nums[x])
            nums[i] = nums[x];
            i += 1;
        }
    }
    return i;
}